sound/soc/sof/sof-audio.c

```diff
@@ -462,6 +503,10 @@ sof_walk_widgets_in_order(struct snd_sof_dev *sdev, struct snd_soc_dapm_widget_l
 		case SOF_WIDGET_UNPREPARE:
 			sof_unprepare_widgets_in_path(sdev, widget);
 			break;
+		case SOF_WIDGET_LARGE_CONFIG:
```

Comment:
How a LARGE_CONFIG is and operation and how much it differs from SETUP operation?

The question is: where do you place the LARGE_CONFIG state in the op flow?

Reply:
LARGE_CONFIG is executed after binding operation.